### DISCUSSION TITLE
Преобразование документов Word docx в нормальный HTML

### DISCUSSION THREAD

**Discussion by erpic-ru** [totumonline/totum-ru-issues-and-discussions#303] [Как это сделать?] [answered]
Нашел оптимальный сервис создания html-шаблонов печатных форм из файлов Word. 
Почти не нужно допиливать руками и тратить нервные клетки.

https://word2cleanhtml.com/

**Comment by ttm-alexeypolunin** [comment]
А запость ссылку отдельным сообщением в этом топике — я его как ответ поставлю и в доки потом ссылку на этот пост сделаю.

**Comment by erpic-ru** [comment, marked as answer]
Собственно вот:
Конвертер в html для word: https://word2cleanhtml.com/

Также шаблоны можно выдернуть из 1С: открыть печатную форму и сохранить в html. Они там немного монструозные, но работают + можно вставить изображения, например, подпись и печать через код.

**Reply by ttm-alexeypolunin** [reply]
Я попробовал — реально круто работает.

Я ток сначала затупил тк не увидел кнопки загрузить файл. Я так понимаю онли открываем док, все выделяем, копируем и в окошко вставляем...
